core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -236,7 +236,9 @@ public void parse(IntsRef ints, ReaderWay way) {
         // Note: if you update this list you have to update the method getAccessValue too
         List<String> accessList = Arrays.asList(
                 //"designated", "permissive", "customers", "delivery",
-                "yes", "destination", "private", "no"
+                "yes", "destination",
+                //"private", <= this currently conflicts with SpatialRule.Access, as this only allows yes, conditional, and no
```

Comment:
I think this shouldn't be an issue? This did not work with the SpatialRule.Access, as this only contains 3 entries. The returned integer was used in the DataFlagEncoderTest like this:

```
idx = accessClassEnc.getInt(false, ints);
assertEquals(SpatialRule.Access.NO, SpatialRule.Access.values()[idx]);
```

Is this the expected behaviour?

Reply:
Both should not depend on each other as there are much more than 2 conditional restrictions we need a mapping here.